ts/parts/Stacking.ts

```diff
@@ -947,3 +956,6 @@ Series.prototype.getStackIndicator = function (
 
     return stackIndicator as any;
```

Comment:
We should try to remove the `any` casts. Maybe the key string can be narrowed down to string literal types?

`stackIndicator` should be type safe, too.

Reply:
I am referring to all the code in the function. Somehow GitHub forgot the marking. 🤷‍♀️